### PR TITLE
Finalized readme and changelog

### PR DESCRIPTION
**Pull request by RuTsEST**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,49 +1,42 @@
...
 ## [Unreleased]
 ### Changed
 - Changed POST /v1/webhooks to accept `application/json` instead of `application/x-www-form-urlencoded` to reflect the reality
 ### Added
-- added required flag for pipeline name in `POST /pipelines` endpoint documentation
+* Support for endpoints added and changed endpoints changed after March 31, 2021
+* Support for multiple instances [issue #176](https://github.com/pipedrive/client-nodejs/issues/176)
+* Convenience functions regarding OAuth 2.0 lifecycle including `buildAuthorizationUrl`, `authorize` and `refreshToken`
+* Support for changing the host URL [issue #228](https://github.com/pipedrive/client-nodejs/issues/228)
+* Support for both snake_case and camelCase
```
Comment (s-pogoda) [inline CHANGELOG.md:17]: Does that apply to the params or response, or both?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,49 +1,42 @@
...
+* `UpdateProductResponse` for `PUT /products/{id}`
+* `Team` (singular) schema for endpoints that deal with a single team
+* Added support for custom fields for `Content-Type: application/x-www-form-urlencoded`
```
Comment (s-pogoda) [inline CHANGELOG.md:20]:
```suggestion
* Support `Content-Type: application/x-www-form-urlencoded` header for custom fields
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,49 +1,42 @@
...
+* Add into documentation that also a Lead is one of the possible returned entities in related objects for the search endpoint.
```
Comment (s-pogoda) [inline CHANGELOG.md:21]: Should it be in the **Changed** section? something like:
- Updated documentation for search endpoint: include Lead as one of the possible returned entities in related objects

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,49 +1,42 @@
...
+* Add note field to call logs.
```
Comment (s-pogoda) [inline CHANGELOG.md:22]: In most cases, we use the past forms for the verbs. What do you think about Added, or better to remove it at all?
```suggestion
* Added note field to call logs.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,49 +1,42 @@
...
-### Changed
 * Deleted `MailThreads` and `MailMessages` from API operation groups and combined them under `Mailbox` group.
 * Extracted `LeadLabels` and `LeadSources` API operation groups from `Leads` group.
-### Added
-* Add note field to call logs.
-### Added
-* Add into documentation that also a Lead is one of the possible returned entities in related objects for the search endpoint.
+### Removed
+* Deleted deprecated `note` field from all lead related documents.
+* Fixed typo in lead example response (`crrency` to `currency`)
```
Comment (s-pogoda) [inline CHANGELOG.md:39]: What do you think about moving it to the **Changed** section, or create **Fixed** one

File: README.md [hunk 6]
```diff
@@ -82,9 +47,9 @@ app.get('/', async (req, res) => {
...
 
 ### With OAuth 2
-If you would like to use OAuth access token for making API calls then make sure the API key,
-that was described in the previous section, is not set or is set to an empty string. If both API token and OAuth access token
-are set, then the API token takes precedence.
+If you would like to use an OAuth access token for making API calls, then make sure the API key described in the previous section is not set or is set to an empty string. If both API token and OAuth access token are both set, then the API token takes precedence.
+
+To set up authentication in the API client, you need the following information. You can receive the necessary client tokens through a Sandbox account (get it [here](https://developers.pipedrive.com/start-here)) and generate the tokens (detailed steps [here](https://pipedrive.readme.io/docs/marketplace-manager#section-how-to-get-your-client-id-and-client-secret)).
 
 In order to setup authentication in the API client, you need the following information.
```
Comment (s-pogoda) [inline README.md:54]: Do we need line 54 as it is duplicating the beginning of line 52?

File: README.md [hunk 7]
```diff
@@ -11,69 +11,34 @@ It provides convenient access to the Pipedrive API, allowing you to operate with
...
-npm run build
-```
-
 ## API Reference
-The Pipedrive RESTful API Reference can be found at https://developers.pipedrive.com/docs/api/v1
+The Pipedrive RESTful API Reference can be found at https://developers.pipedrive.com/docs/api/v1.
+Pipedrive API’s core concepts for its usage can be found in our [Developer documentation](https://pipedrive.readme.io/docs/core-api-concepts-about-pipedrive-api).
 
-## License
-This Pipedrive API client is distributed under the MIT license.
```
[the remark this answers is not in the document]
Comment (RuTsEST) [reply README.md:49]: It is mentioned in the first paragraph so we thought no.

File: CHANGELOG.md (hunk 1 above)
Comment (RuTsEST) [reply CHANGELOG.md:17]: Just params, will clarify in changelo

File: CHANGELOG.md (hunk 2 above)
Comment (RuTsEST) [reply CHANGELOG.md:20]: I think `Support for` is probably best here

File: CHANGELOG.md (hunk 4 above)
Comment (RuTsEST) [reply CHANGELOG.md:22]: Will change it to past form

**Review by s-pogoda — APPROVED** [review]
🏆

**Pull request merged**